### ISSUE TITLE
Assets

### ISSUE THREAD

**Issue by mikesz88** [Kenzie-Tech-Solutions/theRoyalCanine#4]
Ok the client did not give me an exact location of where to put said assets, so I will guesstimate where to put them and then let her tell me if she likes it there or not.

1.  This part on the home page: 
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/2f0cc94c-850f-43d1-834c-89f2903e2d11)
- Let's make this image as a background for the Heading and contact info sections. 
  - That means we need to make the image more transparent and larger height-wise.
  - That also means we might need to make the background of these said sections contrasted with black or with a background that will make it legible
  - I will not be surprised if the client does not like this, but I am trying to conserve space here as the image needs more purpose
2. On the home page, after point one, combining those parts, I want image0 and image7 to be side by side on the desktop/tablet screen and stacked on mobile. 
  - I also want to add the footer text below: "© 2024 The Royal Canine. Website created by [Kenzie Tech Solutions LLC](https://www.kenzietechsolutions.com/)
3. Place all images provided in the PhotoGallery Page.
  - The Photogallery page is not aligned with what the model page. 
  - Basically make it like https://www.shampoochgahanna.com/shampooch-gallery
    - But also keep what you have on the bottom with what you already have.

Link to assets folder: https://drive.google.com/drive/folders/1E0Zb-rXMYdFxiIeHCBvFrGyGMxBBliNW?usp=drive_link

**Comment by Iconians** [comment]
Hi Mike I clicked on the Google Drive link and it told me I was denied access, is the drive the clients or yours and are they expecting a request to access their images from me?
